### PR TITLE
chore(docs): add chart repository docs

### PR DESCRIPTION
**Pull request by michelleN**
#639

### CODE REVIEW

**Comment by technosophos** [comment]
Yay!

File: docs/chart_repository.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Chart Repository Guide
+
+## Prerequistes
+* Go through the [Quickstart](quickstart.md) Guide
+* Read through the [Charts](charts.md) document
+
+## Create a chart repository
+A _chart repository_ is an HTTP server that houses one or more packaged charts. When you're ready to share your charts, the preferred mechanism is a chart repository. You can contribute to the official helm chart repository or create your own. Here we'll talk about creating your own chart repository.
```
Comment (runseb) [inline docs/chart_repository.md]: maybe link [create your own](kubernetes/charts)

other than that LGTM

Comment (michelleN) [inline docs/chart_repository.md]: Thanks @runseb. I'm going to hold off on linking to kubernetes/charts until I have the docs for officially contributing charts is all sorted out, but I'll definitely insert that link in the next few PRs.

**Comment by michelleN** [comment]
awesome. I'll take that ambiguous line out. Thanks!

**Pull request merged**